Sources/ShopifyCheckoutSheetKit/CheckoutWebView.swift

```diff
@@ -152,10 +152,8 @@ extension CheckoutWebView: WKScriptMessageHandler {
 		do {
 			switch try CheckoutBridge.decode(message) {
 			case let .checkoutComplete(checkoutCompletedEvent):
-				CheckoutWebView.cache = nil
 				viewDelegate?.checkoutViewDidCompleteCheckout(event: checkoutCompletedEvent)
 			case .checkoutUnavailable:
-				CheckoutWebView.cache = nil
 				viewDelegate?.checkoutViewDidFailWithError(error: .checkoutUnavailable(message: "Checkout unavailable."))
```

Comment:
Called in the delegate function:
https://github.com/Shopify/checkout-sheet-kit-swift/blob/main/Sources/ShopifyCheckoutSheetKit/CheckoutWebViewController.swift#L179-L182